applications/rag/main.tf

```diff
@@ -169,6 +170,8 @@ module "kuberay-cluster" {
   enable_tpu             = local.enable_tpu
   autopilot_cluster      = local.enable_autopilot
   google_service_account = var.ray_service_account
+  db_secret_name         = module.cloudsql.db_secret_name
```

Comment:
Also update the module in ray

Reply:
Good catch!

Instead, I made all the secrets optional. TIL: https://stackoverflow.com/questions/48222827/optional-volume-secret-volume-in-kubernetes

Tested that /applications/ray deploys successfully and the Ray pods come up.